src/engine/renderer/tr_vbo.cpp

```diff
@@ -305,10 +305,10 @@ static void R_SetVBOAttributeLayouts( VBO_t *vbo )
 }
 
 // index has to be in range 0-255, weight has to be >= 0 and <= 1
-static inline unsigned short
+static unsigned short
 boneFactor( int index, float weight ) {
 	int scaledWeight = lrintf( weight * 255.0 );
```

Comment:
maybe we can reuse `floatToUnorm8`?

https://github.com/DaemonEngine/Daemon/blob/95915040470b774c818b90caec609b7b3dacfa6f/src/engine/renderer/tr_local.h#L53-L57

Reply:
Not going to investigate this since there is a better possibility to use the IQM byte values directly without float conversion.

Reply:
OKOK. That would be great to have full-byte compute without float indeed.